src/ImageSharp/Common/Helpers/ImageMaths.cs

```diff
@@ -13,6 +14,31 @@ namespace SixLabors.ImageSharp
     /// </summary>
     internal static class ImageMaths
     {
+        /// <summary>
+        /// Determine the Greatest CommonDivisor (GCD) of two numbers.
+        /// </summary>
+        public static int GreatestCommonDivisor(int a, int b)
```

Comment:
There was a brief moment a few years back when this method actually existed in the library before! It was used when parsing decimal values for conversion into a Rational.

Reply:
I added LCM, because originally I wanted to optimize `KernelMap`-s buffer size + calculation time further, but all the other issues seemed more critical, so that optimization is postponed for now.